toil-appliance/Dockerfile.template

```diff
@@ -15,8 +15,8 @@ RUN pip install protobuf==3.0.0
 # Install Toil
 RUN pip install toil[aws,mesos,encryption,cwl]==3.3.0
 
-# Point python to where the mesos module is installed
-ENV PYTHONPATH=/usr/lib/python2.7/site-packages/
+# Move the Mesos module onto the python path
+RUN cp -r /usr/lib/python2.7/site-packages/mesos /usr/local/lib/python2.7/dist-packages/mesos
```

Comment:
This was what I originally tested with but thought setting PYTHONPATH was cleaner. However when I tested the new way I only tested if the leader could find the mesos module. It turns out the worker container had problems with that, I think because of the environment manipulation that toil does in the worker.